app/views/users/profile.html.erb

```diff
@@ -153,7 +153,7 @@
   <ul class="nav nav-tabs">
     <span style="float:right;"><small><%= raw t('users.profile.joined_time_ago', :time_ago => distance_of_time_in_words(@user.created_at, Time.current, false, :scope => :'datetime.time_ago_in_words')) %></small></span>
     <li class="active"><a href="#research" data-toggle="tab"><%= t('users.profile.research') %></a></li>
-    <li><a href="#questions" data-toggle="tab"><i class="fa fa-comments"></i><span class="hidden-sm hidden-xs"> <%= t('users.profile.questions') %></span></a></li>
+  <li><a href="#questions" data-toggle="tab"><i class="fa fa-comments"></i><span class="hidden-sm hidden-xs"> <%= t('users.profile.questions') %> (<%= DrupalNode.questions.where(status: 1, uid: @user.id).count %>)</span></a></li>
```

Comment:
Oops -- looks like we ought to have used `length` and not `count` -- probably because of the association we're making. I'll fix this in a followup, but otherwise looks good!